requirements.txt

```diff
@@ -1,10 +1,3 @@
 # The order of packages is significant, because pip processes them in the order
 # of appearance. Changing the order has an impact on the overall integration
 # process, which may cause wedges in the gate later.
```

Comment:
With this change, how is "python setup.py install" going to install these removed dependencies?

I remember we had to remove references to requirements.txt and test-requirements.txt from tox.ini and not actually remove the requirements.txt contents